### DISCUSSION TITLE
Status of Tasks in Meeting Notes

### DISCUSSION THREAD

**Discussion by MucahitErdoganUnlu** [bounswe/bounswe2024group11#120] [Ideas]
I propose that from now on, we do not keep the Status of Tasks in Meeting Notes

**Comment by umitcan07** [comment]
Agreed, starting with the latest meeting notes I put up, let's discard the statuses.

**Comment by MucahitErdoganUnlu** [comment]
@MucahitErdoganUnlu @umitcan07 @ceydanursen @Meminseeker @meminciftci @tenacke @hks1444 @ArdaVural @yunuskaydin @ozankrkya

**Reply by Meminseeker** [reply]
I also totally agree with you. It's a bit useless. Generally, they stay outdated in the meeting notes. Good point!

**Comment by meminciftci** [comment]
Agreed. I will create an issue and handle the deletion of status column on action items matrix of previous meetings if everyone agrees.

@ArdaVural @ozankrkya @ceydanursen @tenacke @hks1444 @yunuskaydin

**Reply by MucahitErdoganUnlu** [reply]
No need. It would be an unnecessary work I believe. Let's not include that column from now on.
